frontend/src/features/admin-form/create/workflow/hooks/useAdminFormWorkflow.ts

```diff
@@ -36,14 +37,28 @@ export const useAdminFormWorkflow = () => {
     [augmentedFormFields],
   )
 
+  const yesNoFormFields = useMemo(
+    () =>
+      augmentedFormFields.filter(
+        (
+          field,
+        ): field is FormFieldWithQuestionNo<FormFieldDto<YesNoFieldBase>> =>
+          field.fieldType === BasicField.YesNo,
+      ),
+    [augmentedFormFields],
+  )
+
+  const formWorkflow =
+    form?.responseMode !== FormResponseMode.Multirespondent
+      ? undefined
```

Comment:
Any particular effects we're trying to achieve with `undefined`? Otherwise would recommend using `null` instead. 

```
const undefinedFoo = undefined;
const nullFoo = null;
console.log(JSON.stringify({ undefinedFoo, nullFoo})) // { nullFoo: null }, undefined will not be present
```

Reply:
changed to 
```
const formWorkflow =
    form?.responseMode === FormResponseMode.Multirespondent
      ? form.workflow
      : null
```!

Reply:
Seems like setting it to `null` breaks the workflow page, investigating

Reply:
it seems if `null` is used instead of undefined, 

```
  const {
    yesNoFormFields = [],
    formWorkflow = [],
    isLoading,
  } = useAdminFormWorkflow()
  ```
  
  no longer sets `formWorkflow` to `[]` when null. 
  hence, i will need to do an explicit check `if(!formWorkflow) formWorkflow = []` to emulate it. 
  
  thinking about it, perhaps `undefined` may be more precise since it is never defined for non-mrf forms instead of defined as `nothing`. hence, i will revert back to `undefined`.